dockerfiles/che/Dockerfile

```diff
@@ -5,7 +5,7 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-FROM openjdk:8u242-jre-slim
+FROM openjdk:11-jre-slim
```

Comment:
I would opt-in for https://pkgs.alpinelinux.org/packages?name=openjdk11-jre&branch=v3.11

to be multi-arch

Reply:
I didn't understand. Could you please rephrase?

Reply:
Something like that 
```
# OpenJDK v11.0.5
# docker run --rm pottava/java:11

FROM alpine:3.11

RUN apk --no-cache add "openjdk11=11.0.5_p10-r0"

ENV JAVA_HOME=/usr/lib/jvm/default-jvm/jre
CMD ["/usr/bin/java", "-version"]
```
?

Reply:
```
openjdk                                                                                        alpine11_jdk11        d5c30a52536c        18 seconds ago      272MB
openjdk                                                                                        11-jre-slim           f39b71460af0        6 days ago          204MB
openjdk                                                                                        11-jre                786611991d70        6 days ago          285MB
openjdk                                                                                        11-jdk                ecea1177bed0        6 days ago          627MB
fabric8/java-alpine-openjdk11-jre                                                              latest                f981116ca5a7        3 weeks ago         187MB
openjdk                                                                                        15-jdk-alpine         97674f54ffe1        4 weeks ago         339MB
openjdk                                                                                        8u242-jre-slim        381b20190cf7        7 weeks ago         184MB
openjdk                                                                                        8u232-jre-slim        5e4f4ec87a1f        3 months ago        184MB
fabric8/java-alpine-openjdk8-jre                                                               latest                6383f0513235        5 months ago        87.3MB
```
272 vs 204

Reply:
@skabashnyuk 
you're comparing a JDK on alpine vs a JRE on Debian

```
FROM alpine:3.11
RUN apk --no-cache add "openjdk11-jre=11.0.5_p10-r0"
```

```
alpine11-jre                    latest                      dba985726831        About a minute ago   186MB
alpine11-jdk                    latest                      4a3df5f079e7        2 minutes ago        272MB
```

Reply:
so it's 186MB + being multi-arch

Reply:
Do you think we can use a single Dockerfile for multi-arch?

Reply:
I would say yes :+1:

Reply:
I was able to get che working with such dockerfile
```
FROM alpine:3.11
RUN apk --no-cache add "openjdk11-jre=11.0.5_p10-r0"  bash
ENV LANG=C.UTF-8
ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
RUN echo "%root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && rm -rf /tmp/*
EXPOSE 8000 8080
COPY entrypoint.sh /entrypoint.sh
ENTRYPOINT ["/entrypoint.sh"]
RUN mkdir /logs /data && \
    chmod 0777 /logs /data
ADD eclipse-che /home/user/eclipse-che
RUN find /home/user -type d -exec chmod 777 {} \;
```
@benoitf  is that ok for you. Changes: 
 - Bash is needed for entrypoint.sh
 -  Explicitly setup `JAVA_HOME`

Reply:
maybe we don't need bash in entrypoint? But could be done separately ( rewrite it to get rid of bash)